feature-utils/poly-look/src/utils/smoothScroll.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * A function to get smooth scrolling behaviour on all browsers (scroll-behaviour:smooth doesn't work in safari)
+ * @param {string} childId - Dom element with ID that is being scrolled to
+ * @param {string} [parentId] - Dom element with ID that is scrollable and parent of child element
+ * @param {string[]} [topNavigationElementIds] - Array of Dom ID's that sit on top of the parent element (usually navigation elements)
+ * @param {Object} [options] - Options regarding the scrolling behaviour
+ * @param {string} [options.speed = 10] - Speed of the scrolling (in ms; smaller = faster)
+ * @param {string} [options.step = 10] - Steps size of the scrolling (smaller = smoother but slower)
+ */
+export function ScrollSmoothly(
```

Comment:
The name is different from the file name, would it be possible to keep them the same?
Also, this is going to be singularly difficult to test, since it includes timing and everything, but it would be nice if some testing was made, for instance checking if the final position is as required, above or below.

Reply:
Yes, I can change the name :)

Regarding the testing, @RichardSto and I tried to add some test to this yesterday, but unfortunately we couldn't make it work because scrolling is not supported by JSDOM.

Reply:
Right. We should introduce some end to end tests... maybe later.